homework-1/cash_register.py

```diff
@@ -0,0 +1,66 @@
+class CashRegister:
+
+    def __init__(self):
+        self.total_items = {}  # {'item': 'price'}
+        self.total_price = 0
+        self.discount = 0
+
+    def add_item(self, item_price: dict):
+        self.total_items.update(item_price)
```

Comment:
The name `item_price` is potentially a little confusing here - something like `item_and_price` might have been clearer based on how I interpret your program as working!